rct229/rulesets/ashrae9012019/section12/section12rule1.py

```diff
@@ -11,35 +14,88 @@ class Section12Rule1(RuleDefinitionListIndexedBase):
-                    USER=True, BASELINE_0=True, PROPOSED=False
+                    USER=False, BASELINE_0=True, PROPOSED=True
                 ),
+                required_fields={
+                    "$.buildings[*].building_segments[*].zones[*].spaces[*].miscellaneous_equipment[*]": [
+                        "power"
+                    ],
+                },
+                manual_check_required_msg=MANUAL_CHECK_REQUIRED_MSG,
             )
 
         def get_calc_vals(self, context, data=None):
-            user_spaces = find_all("$..spaces[*]", context.USER)
-            baseline_spaces = find_all("$..spaces[*]", context.BASELINE_0)
+            rmd_b = context.BASELINE_0
+            rmd_p = context.PROPOSED
+            unexpected_misc_equipment_power = []
+            reduced_misc_equipment_power = []
+            misc_equipment_list_b = find_all(
+                "$.buildings[*].building_segments[*].zones[*].spaces[*].miscellaneous_equipment[*]",
+                rmd_b,
+            )
+            misc_equipment_list_p = find_all(
+                "$.buildings[*].building_segments[*].zones[*].spaces[*].miscellaneous_equipment[*]",
+                rmd_p,
+            )
+            # This assumes that the miscellaneous_equipment all match
+            matched_misc_equipment_list_p = match_lists_by_id(
+                misc_equipment_list_b, misc_equipment_list_p
+            )
+            proposed_baseline_misc_equipment_pairs = zip(
+                misc_equipment_list_b, matched_misc_equipment_list_p
+            )
+
+            for (
+                misc_equipment_b,
+                misc_equipment_p,
+            ) in proposed_baseline_misc_equipment_pairs:
+                misc_equipment_power_b = misc_equipment_b["power"]
+                misc_equipment_power_p = misc_equipment_p["power"]
+                if misc_equipment_power_b > misc_equipment_power_p:
+                    reduced_misc_equipment_power.append(misc_equipment_power_b)
+                elif misc_equipment_power_b < misc_equipment_power_p:
+                    unexpected_misc_equipment_power.append(misc_equipment_power_b)
+
             return {
-                "num_user_spaces": len(user_spaces),
-                "num_baseline_spaces": len(baseline_spaces),
+                "reduced_misc_equipment_power": reduced_misc_equipment_power,
+                "unexpected_misc_equipment_power": unexpected_misc_equipment_power,
+                "compliance_path": data["compliance_path"],
             }
 
+        def manual_check_required(self, context, calc_vals=None, data=None):
+            unexpected_misc_equipment_power = calc_vals[
+                "unexpected_misc_equipment_power"
+            ]
+            compliance_path = calc_vals["compliance_path"]
+            return len(unexpected_misc_equipment_power) == 0 and (
+                compliance_path is None or compliance_path != "CODE_COMPLIANT"
```

Comment:
The `CODE_COMPLIANCE` shall be a schema enum.

At the top of this script,
```python
COMPLIANCE_PATH_TYPE = SchemaEnums.schema_enums["CompliancePathOptions2019ASHRAE901"]
```
Then in here,  you can do:
`COMPLIANCE_PATH_TYPE.CODE_COMPLIANCE`